applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -67,6 +68,10 @@ def __init__(self, Model, settings):
         # Find neighbours
         nodal_neighbour_search.Execute()
 
+        for cond in itertools.chain(self.upper_surface_model_part.Conditions, self.lower_surface_model_part.Conditions):
```

Comment:
better stick to native python, use `zip` here

Reply:
Hi! As I can see [here](https://docs.python.org/3/library/functions.html#zip), zip() needs the two lists to have the same number of elements. This means that if the number of conditions of the upper_model_part and lower_model_part are not the same, some will be missing. 

How can we use the built-in zip() to not miss any condition in case there are a different number of them in the model parts?

Reply:
simply create two independent loops. 
Here the code inside the loop is only 2 lines, so i would say its ok to duplicate it.